lib/cocoapods/downloader/cache.rb

```diff
@@ -156,7 +170,16 @@ def uncached_pod(request)
 
           podspecs.each do |name, spec|
             destination = path_for_pod(request, :name => name, :params => result.checkout_options)
-            copy_and_clean(target, destination, spec)
+
+            # add file lock
```

Comment:
"add file lock" does not add any context to any other programmer as to why a file lock is being used here. If the code is simple and self explanatory then remove this otherwise please expand it to explain why the lock is being used.